projects/core/directives/dropdown/dropdown-position-sided.directive.ts

```diff
@@ -0,0 +1,58 @@
+import {Directive, Inject} from '@angular/core';
+import {WINDOW} from '@ng-web-apis/common';
+import {
+    tuiAsPositionAccessor,
+    TuiPositionAccessor,
+    TuiRectAccessor,
+} from '@taiga-ui/core/abstract';
+import {TuiPoint} from '@taiga-ui/core/types';
+
+import {TUI_DROPDOWN_OPTIONS, TuiDropdownOptions} from './dropdown-options.directive';
+
+const OFFSET = 4;
```

Comment:
why not

```ts
@Input(`tuiDropdownSidedOffset`)
offset = 4;
```

Reply:
Because we don't want it altered, it serves as safe space around host. If somebody wants to change that space they can add/subtract space using `TuiClientRectAccessor`.